.gitlab-ci.yml

```diff
@@ -0,0 +1,56 @@
+stages:
```

Comment:
In this section, you write the stages for your CI pipeline, in order of execution, top to bottom. There are some predefined stages, but you can write any name here (except reserved words) to define your stages.